cfme/containers/provider/openshift.py

```diff
@@ -40,7 +40,7 @@ def configloader(prov_config, prov_key):
         token_creds = OpenshiftProvider.process_credential_yaml_key(
             prov_config['credentials'], cred_type='token')
         return OpenshiftProvider(
-            name=prov_config['name'],
+            name=prov_config['name'].encode('utf-8'),
```

Comment:
If we do this in one place, we should do it throughout, I have oft wanted to create a generic config loader, but didn't yet get around to it